modules/500-upmeter/openapi/config-values.yaml

```diff
@@ -16,6 +16,8 @@ properties:
 
       **CAUTION!** Setting this value to one that differs from the current one (in the existing PVC) will result in disk re-provisioning and data loss.
 
+      **CAUTION!** When migrating Upmeter with local storage to other nodes, the pod will hang in the Pending state. In this case, it will be necessary to save the Upmeter database, delete the old PVC and restart it manually.  Local storage refers to the StorageClass associated with the local volume on the node, not the network storage (for example, StorageClass created by the local-path-provider module).
```

Comment:
```suggestion
      **Caution.** When migrating Upmeter with local storage to other nodes, the pod will hang in the Pending state. In this case, it will be necessary to save the Upmeter database, delete the old PVC and restart the pod manually.  Local storage refers to a StorageClass that is associated not with network storage, but with a local volume on a node (for example, StorageClass created by the local-path-provider module).
```